Bottom: e3f99d9e2a4ed93d8cdf0045c871754c0139d950
Top:    4c02390b79f095399b5e84f953485922d38d79dc
Author: Ian Kent <ikent@redhat.com>
Date:   2013-07-13 14:14:41 +0800

autofs-5.0.7 - add after sssd dependency to unit file

When using sss as a map source autofs can sometimes start before
sssd is ready.


---

diff --git a/samples/autofs.service.in b/samples/autofs.service.in
index 056ab5e..777463d 100644
--- a/samples/autofs.service.in
+++ b/samples/autofs.service.in
@@ -1,6 +1,6 @@
 [Unit]
 Description=Automounts filesystems on demand
-After=network.target ypbind.service
+After=network.target ypbind.service sssd.service
 
 [Service]
 Type=forking
